misc/userscripts/password_fill

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash -e
+help() {
+    blink=$'\e[1;31m' reset=$'\e[0m'
+cat <<EOF
+This script can only be used as a userscript for qutebrowser
+2015, Thorsten Wißmann <edu _at_ thorsten-wissmann _dot_ de>
+In case of questions or suggestions, do not hesitate to send me an E-Mail or to
+directly ask me via IRC (nickname thorsten\`) in #qutebrowser on freenode.
+
+  $blink!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!$reset
+  WARNING: the passwords are stored in qutebrowser's
+           debug log reachable via the url qute:log
+  $blink!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!$reset
+
+Usage: run as a userscript form qutebrowser, e.g.:
+  spawn --userscript ~/.config/qutebrowser/password_fill
+
+Behaviour:
+  It will try to find a username/password entry in the configured backend
+  (currently only pass) for the current website and will load that pair of
+  username and password to any form on the current page that has some password
+  entry field.
+
+EOF
+}
+
+set -o pipefail
+shopt -s nocasematch
+
+if [ -z "$QUTE_FIFO" ] ; then
+    help
+    exit
+fi
+
+error() {
+    local msg="$*"
+    echo "message-error '${msg//\'/\\\'}'" >> "$QUTE_FIFO"
+}
+msg() {
+    local msg="$*"
+    echo "message-info '${msg//\'/\\\'}'" >> "$QUTE_FIFO"
+}
```

Comment:
Hmm - maybe `:message-error` and `:message-info` should be `maxsplit=0` commands so they interpret everything literally and quoting like this is unneeded?